test/testdata/js_substreams/src/lib.ts

```diff
@@ -10,4 +11,12 @@ export default class Substreams {
 			blockHash: blk.id,
 		});
 	}
+	@substreams.handlers.map([ClockSchema], MapResultSchema)
```

Comment:
Why is `ClockSchema` define in the handler? The `map` has no input, which is actually a bit weird, don't think it would be should be valid actually now in JavaScript.

I see it's probably because you define `Clock` input in the `substreams.yaml`. Let's use still a `BlockSchema` as input as we want people to not define Clock and leverage the intrinsics actually.